src/shared/Form/Form.tsx

```diff
@@ -0,0 +1,40 @@
+import Modal from '@material-ui/core/Modal';
```

Comment:
To consider - if we use Form components only in Team.tsx, we can move this file to /pages/Team/components.  
  
In /shared we should hold only really "sharable" components (e.g Table Editor, maybe DatePicker).

Reply:
It's good idea, i created folder components and moved Form there.